ixwebsocket/IXWebSocketTransport.cpp

```diff
@@ -776,34 +823,32 @@ namespace ix
                 _txbuf.erase(_txbuf.begin(), _txbuf.begin() + ret);
             }
         }
-
-        std::lock_guard<std::mutex> lck(_lastSendTimePointMutex);
-        _lastSendTimePoint = std::chrono::steady_clock::now();
     }
 
-    void WebSocketTransport::close()
+    void WebSocketTransport::close(uint16_t code, const std::string& reason, size_t closeWireSize)
     {
         _requestInitCancellation = true;
 
         if (_readyState == CLOSING || _readyState == CLOSED) return;
 
         // See list of close events here:
         // https://developer.mozilla.org/en-US/docs/Web/API/CloseEvent
-        // We use 1000: normal closure.
-        //
-        // >>> struct.pack('!H', 1000)
-        // b'\x03\xe8'
-        //
-        const std::string normalClosure = std::string("\x03\xe8");
+        
+        std::stringstream stream;
+        stream << std::hex << code;
+        const std::string closure(stream.str());
```

Comment:
this is not correct. 
For example for code 1000 it will send text "3e8" instead of "\x03\xe8"

Reply:
Right, I’ll change that